index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>    
+    <nav id="navbar">
+        <a href="#" class="logo">My Logo</a>
+        <ul>
+            <li><a href="#portfolio" class="nav-links">Portfolio</a></li>
+            <li><a href="#About" class="nav-links">About</a></li>
+            <li><a href="#contact" class="nav-links">Contact</a></li>
+        </ul>
+      </nav>
+
+    <section class="first">
+        <div class="container">
+            <div class="boxHead">
+                <h1>
+                    I’m Abdo Zayan<br>
+                    Glad to see you!
+                </h1>
+                <p>
+                    I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
```

Comment:
- [ ] Very Good work on your `desktop version`  but your `mobile version` does not appear to be styled at all. Kindly note that in this milestone you are supposed to come up with the desktop version of your portfolio without interfering with the mobile version.

![image](https://user-images.githubusercontent.com/98453979/236247664-96d16493-998a-4a3c-87ea-124af2598805.png)

Reply:
@frank1738 
mobile styling added